src/Microsoft.DotNet.Wpf/src/PresentationCore/GlobalSuppressions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "Public API", Scope = "member", Target = "~F:System.Windows.DataFormats.Text")]
```

Comment:
Why we are suppressing all these CodeAnalysis messages?

Reply:
Because they are existing public API that violate guidelines. Putting the suppressions here instead of inline makes the code easier to read.